File: others/db/no-sql/java-springboot-helloworld-db-nosql-neo4j-crud/src/main/java/com/example/dtos/HelloWorldDto.java
package com.example.dtos;

import com.example.documents.HelloWorldDoc;

public class HelloWorldDto {
		
    private Long id; 
    private String text;    
	
	public HelloWorldDto() {}

	public HelloWorldDto(HelloWorldDoc doc) {
		this.id = doc.getId();
		this.text = doc.getText();
	}
	
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}

	public String getText() {
		return text;
	}
	public void setText(String text) {
		this.text = text;
	}

}

File: others/services/fe-angular-be-springboot-db-sql-mysql/fe-angular-be-springboot-db-sql-mysql_FE/src/app/dtos/hello-world.dto.ts
export class HelloWorldDto {

    id: number;
    text: string;
    portBe: string;

    constructor(id: number, text: string, portBe: string) {
        this.id = id;
        this.text = text;
        this.portBe = portBe;
    }
    
}